CONTRIBUTING.md

```diff
@@ -63,6 +63,9 @@ If you have previously run a project on a different port, a `.foreman` file
 may be generated at the root of your directory. If so, make sure that this
 file is set to port `5000` or you will be unable to authenticate locally with MyUSA.
 
+### Testing
+Testing is done using `capybara-poltergeist`, which requires a local install of [phantomjs](https://github.com/jonleighton/poltergeist#installing-phantomjs).
```

Comment:
glad we are adding a section on testing to the docs! great idea. 

In addition to this line about installing poltergeist, how about adding something like:

You can run the entire test suite by running `rake`